romanisim/l3.py

```diff
@@ -459,7 +459,7 @@ def get_pixscalefrac(wcs, shape):
     return pixscale
 
 
-def simulate_cps(image, filter_name, efftimes, objlist=None, psf=None,
+def simulate_cps(image, filter_name, sca, efftimes, objlist=None, psf=None,
```

Comment:
I think my preference is to remove sca from the docstring and to use parameters.default_sca for the L3 code.  But if you keep it, make it default None, and set it to the default_sca if it's found to be none, and add it to the docstring.